### PR TITLE
feat: add a UIImage next to the answer UIButton

### PR DESCRIPTION
**Pull request by stefaniamak**
### Tasks
- [x] set ColumnLayout for the UIImages

### Purpose

Set an arrow asset to appear in the appropriate UIImage next to the hoved UIButton.

### How to test

- Check https://github.com/Terasology/LightAndShadowResources/pull/53
- Check https://github.com/Terasology/Dialogs/pull/15
- Start a LAS game
- Talk to The Fool NPC
- See the image/s next to the answer button/s

_note: the extra response and large text at the dialogue box is not committed and is not planned to be_
image

### CODE REVIEW

**Review by skaldarnar — CHANGES_REQUESTED** [review]
As described in https://github.com/Terasology/Dialogs/pull/15#pullrequestreview-457797353 the actual image to display should be configured via the `MagicFool.prefab`.

**Comment by stefaniamak** [comment]
> 
> 
> As described in [Terasology/Dialogs#15 (review)](https://github.com/Terasology/Dialogs/pull/15#pullrequestreview-457797353) the actual image to display should be configured via the `MagicFool.prefab`.

Yes! Not 100% sure if we need the extra column or not in the end, but if we end up not needing it then this PR can close.

File: assets/prefabs/MagicFool.prefab [hunk 1]
```diff
@@ -31,6 +31,7 @@
                 "responses": [
                     {
                         "text": "Very well..",
+                        "responseImage": "LightAndShadowResources:heartsIcon",
```
Comment (skaldarnar) [inline assets/prefabs/MagicFool.prefab]: 👍

**Review by Cervator — COMMENTED** [review]
Have tested this out along with the two other related PRs - seems fine! 👍 

Clearly a fair bit of back-and-forth review cycles have happened with a bunch of commits with relatively few changes, so I'm going to opt for squash merging these. Changes seem relatively straight forward and nothing really jumps out at me :-)

Nicely done!

**Pull request merged**